cv32e20/bsp/crt0.S

```diff
@@ -70,3 +70,11 @@ _fini:
 	ret
 .size  _init, .-_init
 .size _fini, .-_fini
+
```

Comment:
It would be useful to have a couple of comment lines explaining what the `tohost` section is for and where this code places it in the memory space.